### DISCUSSION TITLE
Notes are formatted weird on desktop

### DISCUSSION THREAD

**Discussion by EvilCartyen** [gramps-project/gramps-web#514] [General]
When you create a note for a person or other entity it looks alright on mobile, but when displayed on wider screens things look weird as even limited note content is split into several columns:
![image](https://github.com/user-attachments/assets/92b71117-bc72-4c08-a198-437045f0ae46)

**Comment by DavidMStraub** [comment]
This is simply CSS with a maximum column width. I don't know if CSS allows to specify a minimum number of rows per column.

**Comment by EvilCartyen** [comment]
Well why do we need more columns? Wouldnt a single column suffice?

**Comment by EvilCartyen** [comment]
I don't know much about frontend, but setting the note element to 100 em or so seems to make everything look nice regardless of the screen size.

**Comment by DavidMStraub** [comment]
I'll convert this to a discussion as it's a matter of style/taste. Not saying I disagree with you, but I think there's no need to track it as a bug.

**Comment by DavidMStraub** [comment]
https://github.com/gramps-project/gramps-web/commit/bbd311c36542bd2e66e47f530dc2921ed429793d
